Fabulous.CodeGen/src/Fabulous.CodeGen/Binder/Binder.fs

```diff
@@ -272,11 +272,18 @@ module Binder =
     /// Bind an existing type
     let bindType logger (assemblyType: AssemblyType) shouldGenerateBindingForType (bindingsType: Type) =
         let typeName = BinderHelpers.getTypeName assemblyType.FullName bindingsType.Name
+        let constructorParameters =
+            assemblyType.ConstructorsParameters
+            |> Array.tryFind (fun x -> bindingsType.ConstructorParameters = x)
```

Comment:
`ConstructorsParameters` contain all available constructors for the type. `ConstructorParameters` contain required constructor parameters for type. By default, the array is empty, so, `CanBeInstantiated` will always false if `ConstructorParameters` is not set and there is no constructor without parameters.